packages/material-ui/src/Box/Box.d.ts

```diff
@@ -14,9 +15,425 @@ import {
   PropsFor,
 } from '@material-ui/system';
 import { Theme } from '../styles/createMuiTheme';
-import { CSSObject } from '../styles/experimentalStyled';
 import { Omit } from '..';
 
+/**
+ * CSS typings
+ */
+
+export type StandardCSSProperties = CSS.PropertiesFallback<number | string>;
+
+/**
+ * The `css` function accepts arrays as values for mobile-first responsive styles.
+ * Note that this extends to non-theme values also. For example `display=['none', 'block']`
+ * will also works.
+ *
+ * For more information see: https://styled-system.com/responsive-styles
+ */
+export type ResponsiveStyleValue<T> = T | Array<T | null> | { [key: string]: T | null };
+
+/**
+ * All non-vendor-prefixed CSS properties. (Allow `number` to support CSS-in-JS libs,
+ * since they are converted to pixels)
+ */
+export interface CSSProperties
+  extends CSS.StandardProperties<number | string>,
+    CSS.SvgProperties<number | string> {}
+
+/**
+ * Map of all CSS pseudo selectors (`:hover`, `:focus`, ...)
+ */
+export type CSSPseudoSelectorProps = { [K in CSS.Pseudos]?: SystemStyleObject };
+
+/**
+ * CSS as POJO that is compatible with CSS-in-JS libaries.
+ * Copied directly from [emotion](https://github.com/emotion-js/emotion/blob/ca3ad1c1dcabf78a95b55cc2dc94cad1998a3196/packages/serialize/types/index.d.ts#L45) types
+ */
+export interface CSSObject
+  extends CSSPropertiesWithMultiValues,
+    CSSPseudosForCSSObject,
+    CSSOthersObjectForCSSObject {}
+
+export type CSSPropertiesWithMultiValues = {
+  [K in keyof CSSProperties]: CSSProperties[K];
+};
+export type CSSPseudosForCSSObject = { [K in CSS.Pseudos]?: CSSObject };
+export type CSSInterpolation = undefined | number | string | CSSObject;
+export interface CSSOthersObjectForCSSObject {
+  [propertiesName: string]: CSSInterpolation;
+}
+
+/**
+ * Map all nested selectors
+ */
+export interface CSSSelectorObject {
+  [cssSelector: string]: SystemStyleObject;
+}
+
+export interface AliasesCSSProperties {
+  /**
+   * The **`background-color`** CSS property sets the background color of an element.
+   *
+   * **Initial value**: `transparent`
+   *
+   * | Chrome | Firefox | Safari |  Edge  |  IE   |
+   * | :----: | :-----: | :----: | :----: | :---: |
+   * | **1**  |  **1**  | **1**  | **12** | **4** |
+   *
+   * @see https://developer.mozilla.org/docs/Web/CSS/background-color
+   */
+  bgcolor?: StandardCSSProperties['backgroundColor'];
```

Comment:
The comments are done in this format, so they will be consistent with the intellisense for the other props which comes from the `csstype` library.

Reply:
Yeah, but we can't change the descriptions for the props already coming from `csstype`. We can however not include the table from the props we are overriding. Should we do that?

Reply:
Let's see what Sebastian and Matt think.

Reply:
Better to be consistent I would have thought?

Reply:
I'm all in for consistency when the value is about having a single convention, when one way or the other is more or less the same. In this case, it seems that this table is a distraction with zero value. What do you gain from the consistency in this case? Anyway, no strong preferences.

Reply:
Alright, will resolve it then